inst/examples/router/app.r

```diff
@@ -0,0 +1,90 @@
+library(shiny)
```

Comment:
This does not work but was an attempt to implement: https://framework7.io/docs/view with shiny.router. I also tried with `brochure` long time ago.

Reply:
@hypebright Now this works better with `{brochure}` (not on CRAN). However this does not prevent us from writing a vignette on how to make it possible. This is pretty straightforward after all. Perhaps @ColinFay will like to put `{brochure}` on CRAN at a later point.

Reply:
@DivadNojnarg yes we're doing more vignettes in this version anyway, like about inputs in a list and reconnect and such (and update all of the existing ones) so would be great to have one about this solution as well. It's really cool functionality!